nixos/modules/services/misc/navidrome.nix

```
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.navidrome;
+  configFile = "/etc/navidrome/settings.json";
+  dataFolder = "/var/lib/navidrome";
+in {
+  options = {
+
+    services.navidrome = {
+      enable = mkEnableOption "Enable Navidrome music server and streamer";
```

```
  mkEnableOption =
    # Name for the created option
    name: mkOption {
    default = false;
    example = true;
    description = "Whether to enable ${name}.";
    type = lib.types.bool;
  };
```

"Enable" is redundant.

---

I read the nixpkgs manual but I can't understand this usage, could you elaborate, please?

---

Sure. Specifically the `description` provided already has the text "Whether to enable" in it. So when you write `mkEnableOption "Enable Navidrome music server and streamer"` it comes out as "Whether to enable Enable Navidrome music server and streamer.".
```suggestion
      enable = mkEnableOption "Navidrome music server and streamer";
```

---

Ok, so I simply removed "Enable" in the string.